src/pathfinding_service/api.py

```diff
@@ -33,12 +34,14 @@
 from raiden_libs.types import Address, TokenNetworkAddress
 
 log = structlog.get_logger(__name__)
+# change maxlen in case more than 20 paths are requested
+last_requests: collections.deque = collections.deque([], maxlen=20)
 
 
 class ApiWithErrorHandler(Api):
     def handle_error(self, e: exceptions.ApiException) -> Response:
         return self.make_response(
-            {'errors': e.msg, 'error_code': e.error_code, 'error_details': e.error_details},
+            {"errors": e.msg, "error_code": e.error_code, "error_details": e.error_details},
```

Comment:
According to the Raiden contributor's guidelines this should be single quotes and not double quotes: https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md

Reply:
I heard raiden is going to use black. If that's true, that part of the code style will probably change, since black prefers double quotes: https://github.com/ambv/black#strings .

I mostly don't mind, so don't count this as a vote for any of the choices.

Reply:
Yes this is black preferring the double quotes. I don't care tbh. can we maybe change the black settings somehow @karlb ?

Reply:
Right now, we are running black with `--skip-string-normalization` which means it does not change the quotes. I don't think there is a switch to prefer single quotes due to black's general "any color you want -- as long as it's black"-attitude.

Reply:
Given these changes in PRs, should we just switch to "real" black and remove the `--skip-string-normalization` option?